DigitalLearningSolutions.Web/Views/Signposting/LinkLearningHubSso.cshtml

```diff
@@ -1,11 +1,9 @@
 ﻿@using DigitalLearningSolutions.Web.Helpers
-@using DigitalLearningSolutions.Web.ViewModels.SignpostingSso
+@using DigitalLearningSolutions.Web.ViewModels.Signposting
 @model LinkLearningHubViewModel
 
 @{
-    ViewData[LayoutViewDataKeys.Title] = "Recommended Learning Resources - Account linked";
-    ViewData[LayoutViewDataKeys.Application] = "Recommended Learning Resources";
-    ViewData[LayoutViewDataKeys.HeaderPrefix] = "";
+    ViewData[LayoutViewDataKeys.Title] = "Learning Hub Account Linked";

```

Comment:
Just wondering about the capitalization here, I think we usually only capitalize the first letter in titles so maybe "Learning Hub account linked"?

Reply:
Funnily that was what I was going to go with until I saw this style elsewhere such as:
https://github.com/TechnologyEnhancedLearning/DLSV2/blob/master/DigitalLearningSolutions.Web/Views/Signposting/LearningHubLoginWarning.cshtml#L6
Jumping around the site, it looks like there is some inconsistency on the approach here.
I will update this one as per your recommendation.

Reply:
Thats done now